modules/deployments-comparing-deploymentconfigs.adoc

```diff
@@ -5,25 +5,33 @@
 [id='deployments-comparing-deploymentconfigs-{context}']
 = Comparing Deployments and DeploymentConfigs
 
-Because deployment configurations existed in {product-title} prior to
```

Comment:
why don't we use `{product-title}` var anymore?

Reply:
Good catch, I should have.